AdobeStockImageAdminUi/view/adminhtml/ui_component/adobe_stock_images_listing.xml

```diff
@@ -53,6 +53,13 @@
                     <dataScope>orientation_filter</dataScope>
                 </settings>
             </filterSelect>
+            <filterSelect name="premium_price_filter" provider="${ $.parentName }">
+                <settings>
+                    <options class="Magento\AdobeStockImageAdminUi\Ui\Component\Listing\Columns\PremiumPrice\Options"/>
```

Comment:
Please, add `<caption translate="true">Select...</caption>` for the default filter state

Reply:
I remove it pointedly because I had understood that the default is ALL: ("User sees the default filter "All" applied. It doesn't filter search result")

Reply:
As far as I know, you will not be able to reset the filter to default state without the caption. If you click 'Reset All' button, your filter will be still applied but with 'All' option